File: Recursion I/Pascals_Triangle_II.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        arr = []
        arr.append([1])
        arr.append([1,1])
        for i in range(1,rowIndex):
            l = []
            l.append(1)
            for j in range(len(arr[i])-1):
                l.append(arr[i][j]+arr[i][j+1])
            l.append(1)
            arr.append(l)
        return arr[rowIndex]

File: Recursion I/Pascals_Triangle_II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<vector<int>> v;
        vector<int> l;
        l.push_back(1);
        v.push_back(l);
        l.push_back(1);
        v.push_back(l);
        for(int i = 1;i<rowIndex;i++)
        {
            vector<int>l1;
            l1.push_back(1);
            for(int j = 0; j<v[i].size()-1;j++)
                l1.push_back(v[i][j] + v[i][j+1]);
            l1.push_back(1);
            v.push_back(l1);
        }
        return v[rowIndex];
    }
};